src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -0,0 +1,358 @@
+    },
+  } as CreateOnboardingQuoteInput
+}
+
+const ButtonLoadingIndicator = styled(LoadingDots)`
+  display: inline-flex;
+  margin-left: 0.5rem;
+`
+
+export const QuoteData: React.FC<OfferProps> = ({ sessionId }) => {
+  const { data, refetch } = useOnboardingSessionQuery({
+    variables: { id: sessionId },
+  })
+  const [createOnboardingQuote] = useCreateOnboardingQuoteMutation()
+
+  const currentLocale = useCurrentLocale()
+  const currentMarket = useMarket()
+
+  const [quoteType, setQuoteType] = useState(
+    quotesByMarket[currentMarket][0].value,
+  )
+
+  const [submitError, setSubmitError] = useState<Error>()
+  const handleSubmit = async (values: Values) => {
+    const input = {
+      ...values,
+      currentInsurer: values.currentInsurer || undefined,
+      startDate: values.startDate || undefined,
+    }
+
+    try {
+      switch (quoteType) {
+        case QuoteType.DanishHomeAccident:
+          await Promise.all([
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: input as CreateOnboardingQuoteInput,
+              },
+            }),
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: getDanishQuoteValues(input, 'danishAccidentData'),
+              },
+            }),
+          ])
+          break
+
+        case QuoteType.DanishHomeAccidentTravel:
+          await Promise.all([
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: input as CreateOnboardingQuoteInput,
+              },
+            }),
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: getDanishQuoteValues(input, 'danishAccidentData'),
+              },
+            }),
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: getDanishQuoteValues(input, 'danishTravelData'),
+              },
+            }),
+          ])
+          break
+
+        case QuoteType.SwedishApartmentAccident:
+          await Promise.all([
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: input as CreateOnboardingQuoteInput,
+              },
+            }),
+            createOnboardingQuote({
+              variables: {
+                id: sessionId,
+                input: getSwedishAccidentQuoteValues(input),
+              },
+            }),
+          ])
+          break
+
+        default:
+          await createOnboardingQuote({
+            variables: {
+              id: sessionId,
+              input: input as CreateQuoteVariables['input'],
+            },
+          })
+      }
+
+      await refetch()
+    } catch (error) {
+      setSubmitError(error)
+    }
+  }
+
+  if (submitError) {
+    throw submitError
+  }
```

Comment:
It seems like Sentry catches all errors and just logs them to the console which isn't super helpful when you want to use error boundaries. That's why I had to come up with this workaround.